src/Servers/Connections.Abstractions/src/Features/IStreamDirectionFeature.cs

```diff
@@ -3,10 +3,9 @@
 
 namespace Microsoft.AspNetCore.Connections.Features
 {
-    public interface IQuicStreamFeature
+    public interface IStreamDirectionFeature
```

Comment:
I didn't know if we wanted these to be on the StreamContext yet.

Reply:
I don't think StreamContext is different enough from ConnectionContext to justify its existence. Maybe this could be added to ConnectionContext with both defaulting to true, but as with StreamId, I'm not sure it's useful enough deserve be a top level property on ConnectionContext.